### PR TITLE
test: RUN-959: Remove the scalability benchmarking suite

### PR DESCRIPTION
**Pull request by ulan**
The benchmarking suite was deprecated and will be replaced with a new suite that uses the standard execution testing infrastructure.

### CODE REVIEW

**Comment by ulan** [comment]
See also https://github.com/dfinity/ic/pull/526

File: scalability/run-p8s.sh [hunk 1]
```diff
@@ -1,190 +0,0 @@
-#! /usr/bin/env nix-shell
```
Comment (basvandijk) [inline scalability/run-p8s.sh:1]: Can `scalability/run-p8s.sh` and `scalability/run-p8s.nix` be moved to `/rs/tests/` and owned by `@dfinity/idx`? Some folks are still using these scripts to look at p8s data from a passed system-test run.

**Review by basvandijk — APPROVED** [review]

File: scalability/run-p8s.sh (hunk 1 above)
Comment (ulan) [reply scalability/run-p8s.sh:1]: Sure, I'll move them

Comment (ulan) [reply scalability/run-p8s.sh]: I moved the scripts to `/rs/tests`. I see that that folder is already owned by the IDX team, so I didn't add new entries in the code owners file.

Comment (basvandijk) [reply scalability/run-p8s.sh]: @ulan you need an extra `dirname` [here](https://github.com/dfinity/ic/blob/9e9e7ed6640c58cbbeda0ea9927f69982bf5daa0/rs/tests/run-p8s.sh#L10)

Comment (ulan) [reply scalability/run-p8s.sh]: Thanks! I simplified the expression now that the script is in `/rs/tests`.

**Review by alexandru-uta — APPROVED** [review]
LGTM, thanks!

**Pull request merged**